scripts/cast-on-tenderly/index.js

```diff
@@ -60,22 +119,27 @@ const runSpell = async function () {
     await provider.send('tenderly_setStorageAt', [
         chiefAddress,
         ethers.utils.hexZeroPad(ethers.utils.hexValue(CHIEF_HAT_SLOT), 32),
-        ethers.utils.hexZeroPad(SPELL_ADDRESS, 32),
+        ethers.utils.hexZeroPad(spellAddress, 32),
     ]);
 
     console.info('checking the hat...');
     const chief = new Contract(chiefAddress, ['function hat() external view returns (address)'], signer);
     const hatAddress = await chief.hat();
-    if (hatAddress !== SPELL_ADDRESS) {
+    if (hatAddress.toLowerCase() !== spellAddress.toLowerCase()) {
```

Comment:
I believe it would be better if we ensured `spellAddress` is in the checksummed format before passing it to this function instead of making that conversion.
We know for sure that `hatAddress` will be in the correct format because ethers.js returns it in the right format. 
```suggestion
    if (hatAddress !== spellAddress) {
```